pymc/tests/test_distributions_random.py

```diff
@@ -1163,6 +1163,26 @@ class TestDirichlet(BaseTestDistribution):
     ]
 
 
+class TestStickBreakingWeights(BaseTestDistribution):
```

Comment:
Is there any obvious statistical property you could easily test to be more confident the rng_fn is returning valid samples?

The values must some to 1 right? Maybe other things as well?

Reply:
Yes values must sum to one. Should I also write a test that checks `0 <= value <= 1`?